src/graph/test/SchemaTest.cpp

```diff
@@ -171,62 +200,88 @@ TEST_F(SchemaTest, metaCommunication) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    // Test existent tag
     {
         cpp2::ExecutionResponse resp;
-        std::string query = "CREATE TAG account(id int, balance double)";
+        std::string query = "CREATE TAG person(id int, balance double)";
         auto code = client->execute(query, resp);
-        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```

Comment:
Do we have other return code?
 such as ALREAD_EXISTED.
!= SUCCEED is not precise.